app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt

```diff
@@ -149,6 +150,10 @@ class AppVersionActivityTest {
     }
   }
 
+  private fun PackageManager.getVersionName(): String {
+    return getPackageInfo(context.packageName, 0).versionName
```

Comment:
```suggestion
    return getPackageInfo(context.packageName, /* flags= */ 0).versionName
```

To provide context to readers.

Reply:
Done